Content.Server/GameObjects/Components/GUI/HandsComponent.cs

```diff
@@ -467,6 +468,11 @@ public void RemoveHand(string name)
 
             OnItemChanged?.Invoke();
 
+            if (Owner.TryGetComponent<CuffableComponent>(out var cuffable))
```

Comment:
😬 big yikes on this

Should probably just be a component message, that `CuffableComponent` can then listen to.

Check out the `EventBus`.

Reply:
Something like:
```cs
    public class HandCountChangedEvent : EntitySystemMessage
    {
        public HandCountChangedEvent()
        {
        }
    }
```

You'd have this here (the line of this comment) in `RemoveHand()`:
```cs
Owner.EntityManager.EventBus.RaiseEvent(EventSource.Local, new HandCountChangedEvent());
```

And anything that cares about this (e.g. `CuffableComponent`) would do:
```cs
Owner.EntityManager.EventBus.SubscribeEvent<HandCountChangedEvent>(EventSource.Local, this, HandleHandCountChange);

...

private void HandleHandCountChange(HandCountChangedEventmessage)
{
	if (message.Entity != Owner)
	    return;
        ... //Do something now you know about the change
}

```